admin/customers.php

```diff
@@ -1700,7 +1700,7 @@ class="btn btn-default"><?php
         $additional_columns = [];
         $zco_notifier->notify(
             'NOTIFY_ADMIN_CUSTOMERS_LISTING_ELEMENT',
-            $result,
+            array_merge($result, $customer),
             $additional_columns,
             $customer
```

Comment:
If we're merging $customer with $result, I guess we can remove the (recent) addition of $customer as well, right?

Reply:
I kept that there, since there might be usage of that additional parameter ... no harm in keeping it, IMO.